starboard/extension/ifa.h

```diff
@@ -46,9 +48,37 @@ typedef struct StarboardExtensionIfaApi {
   // In Starboard 14 this the value is retrieved through the system
   // property `kSbSystemPropertyLimitAdTracking` defined in
   // `starboard/system.h`.
-
   bool (*GetLimitAdTracking)(char* out_value, int value_length);
 
+  // The fields below this point were added in version 2 or later.
+
+  // Returns the the user's authorization status for using IFA in this app.
+  // Valid strings that can be returned are:
+  //   * UNKNOWN - if the system isn't able to determine a status
```

Comment:
Late comment after submission but - could we perhaps add "NOT_SUPPORTED" value here, and then remove the `COBALT_ENABLE_EXTENDED_IFA` compile-time config entirely, making the API always available ?